Update マニュアル.md

andam0202 opened the pull request:
Translated the current README.md so that native Japanese speakers can easily understand it.

- Translated only the manual.md directly under the repository, not the manual.md in the doc folder.
- Titles of games sold in Japan have been changed to their Japanese names.
- The links to the games were not changed to the Japanese sites, but left as they are.

This is my first pull request, so sorry if I'm wrong.
Ask me questions if you still find something wrong.

Sounds like an interesting project!
I'm willing to help out a little bit.
If there are any major changes in the README.md I would like to translate it myself.

andam0202 committed:
Update マニュアル.md

Translated the current README.md so that native Japanese speakers can easily understand it.

- Translated only the manual.md directly under the repository, not the manual.md in the doc folder.
- Titles of games sold in Japan have been changed to their Japanese names.
- The links to the games were not changed to the Japanese sites, but left as they are.

Sauceke commented:
I probably won't be able to keep this file up to date in the long run, but this looks great!

Sauceke commented on マニュアル.md:
```diff
@@ -1,51 +1,203 @@
+[LoveMachine.Experiments]プラグインは、[コイカツ！]と[コイカツ！サンシャイン]で実験的な2つの深度センシングデバイス[Lovense Calor]と[Hotdog]のサポートを追加します。
+
+## インストール
+Download and run the [installer]. If you encounter the "Windows protected your PC" message, click More info > Run anyway.
```
Do you need help in translating this part?
I don't have access to a Windows machine right now, but this is what the message looks like in English: https://i.stack.imgur.com/3t0tg.jpg

Sauceke commented on マニュアル.md:
```diff
@@ -1,51 +1,203 @@
+[LoveMachine.Experiments]プラグインは、[コイカツ！]と[コイカツ！サンシャイン]で実験的な2つの深度センシングデバイス[Lovense Calor]と[Hotdog]のサポートを追加します。
+
+## インストール
+Download and run the [installer]. If you encounter the "Windows protected your PC" message, click More info > Run anyway.
```
This line is no longer needed, as you have translated it below.

andam0202 replied:
The relevant section has been deleted.

Sauceke commented on マニュアル.md:
```diff
@@ -1,51 +1,203 @@
+
+スペースキーは、ゲーム中にすべてのデバイスを停止する停止ボタンとして機能します。デバイスを再起動するにはF8キーを推してください。これらのキー割り当ては、プラグイン設定 > LoveMachine > Kill Switch Settingsで変更できます。
+
+⚠ In certain games, the kill switch may not work if the BepInEx console is open while playing in VR, because it can steal focus from the game window. It is recommended to disable the console.
```
This line is no longer needed, as you have translated it below.

andam0202 replied:
The relevant section has been deleted.

Sauceke commented on マニュアル.md:
```diff
@@ -1,51 +1,203 @@
-## 使い方
+このMODは無料ですが、寄付を受け付けています。私の開発を応援してくださる方は、[こちら]からよろしくお願いします。
+### 開発者
+Sauceke   •   nhydock   •   hogefugamoga   •   RPKU
```
You are contributing to the code base, so your name should go here.

andam0202 replied:
Thank you very much. I am afraid I will add my name there.

Sauceke commented:
This file is the "live" (deployed) version of the readme, which is overwritten with ``doc/マニュアル.md`` on each release.
Please also overwrite ``doc/マニュアル.md`` with an identical copy of this file.

andam0202 committed:
fix マニュアル.md

I have updated the manual.md in the doc folder to correct the areas you pointed out.

Sauceke commented on doc/マニュアル.md:
```diff
@@ -1,51 +1,196 @@
-現在互換性があるデバイスは：
+LoveMachineはアダルトゲームのキャラクターと連動してアダルトグッズを動かすプラグインです。 
+
+[一部の自律型デバイス](#supported-devices)について、以下のゲームに対応しています。
```
Broken link
```suggestion
[一部の自律型デバイス](#サポートしているデバイス)について、以下のゲームに対応しています。

```

Sauceke commented on マニュアル.md:
```diff
@@ -1,51 +1,196 @@
-現在互換性があるデバイスは：
+LoveMachineはアダルトゲームのキャラクターと連動してアダルトグッズを動かすプラグインです。 
+
+[一部の自律型デバイス](#supported-devices)について、以下のゲームに対応しています。
```
Broken link
```suggestion
[一部の自律型デバイス](#サポートしているデバイス)について、以下のゲームに対応しています。

```

andam0202 committed:
Fix "supported-devices" link

Sauceke commented:
I haven't merged it yet ^^

Sauceke commented:
Thank you, your changes have now been deployed.
